bin/hdfcoinc/pycbc_plot_psd_file

```diff
@@ -128,4 +132,5 @@ pycbc.results.save_fig_with_metadata(fig, args.output_file,
     caption="Median amplitude spectral density plotted with a shaded region " 
               "between the 5th and 95th perentiles. ",
     cmd=' '.join(sys.argv),
-    fig_kwds={'dpi': 200})
+    fig_kwds={'dpi': 200,
+              'bbox_inches': 'tight'})
```

Comment:
Is there a reason that these don't get set along with the style?

Reply:
I figured what you want to set for these might be program specific.